spec/omniauth/strategies/auth0_spec.rb

```diff
@@ -25,13 +25,6 @@
     )
   end
 
-  around do |t|
-    allowed_request_methods = OmniAuth.config.allowed_request_methods
```

Comment:
why are these tests removed?

Reply:
This is removing a workaround that was introduced in https://github.com/auth0/omniauth-auth0/pull/116 because of failing tests being caused by `omniauth` not being pinned